src/uiSelectMultipleDirective.js

```diff
@@ -258,7 +258,7 @@ uis.directive('uiSelectMultiple', ['uiSelectMinErr','$timeout', function(uiSelec
 
         if(caretPosition > 0 || ($select.search.length && key == KEY.RIGHT)) return false;
 
-        $select.close();
+        $select.close(null, true);
```

Comment:
Yeah I really with I could use named parameters there too, but I didn't find a way to do it that didn't make me oh so sad 😿